Assets/Scripts/AtmosphericAsteroidThrustCapacityEffect.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using UnityEngine;
+
+public class AtmosphericAsteroidThrustCapacityEffect : ThrustCapacityEffectBehavior
```

Comment:
This seems a bit over-engineered...  why do we need this class instead of just an `AtmosphericAsteroidBehavior` script that calls a `Refuel(int amount)` method on `ThrusterManager`? I really don't think we'll need `ThrustCapacityEffectBehavior` for anything other than this one use case.

Reply:
Aight